evap/results/templates/results_semester_detail.html

```diff
@@ -34,21 +34,18 @@ <h3>{{ semester.name }}</h3>
                             </thead>
                             <tbody>
                             {% for course in courses %}
-                                <tr {% if course.state == 'in_evaluation' or course.state == 'evaluated' or course.state == 'reviewed' %}class="preview"{% endif %}>
+                                <tr class="{% if not course|can_user_see_results:request.user %}nohover{% endif %}{% if course.state == 'in_evaluation' or course.state == 'evaluated' or course.state == 'reviewed' %} preview{% endif %}" {% if course|can_user_see_results:request.user %} data-url="{% url 'results:course_detail' semester.id course.id %}"{% endif %}>
                                     <td data-order="{{ course.name }}">
                                         {% if course.state == 'in_evaluation' %}
                                             <span data-toggle="tooltip" data-placement="top" class="fa fa-play" title="{% trans 'This course is still in evaluation.' %}"></span>
                                         {% endif %}
                                         {% if course.state == 'evaluated' or course.state == 'reviewed' %}
                                             <span data-toggle="tooltip" data-placement="top" class="fa fa-pause" title="{% trans 'This course has not been published yet.' %}"></span>
                                         {% endif %}
-                                        {% if course|can_user_see_results:request.user %}
-                                            <a href="{% url "results:course_detail" semester.id course.id %}">{{ course.name }}</a>
-                                        {% else %}
-                                            <span data-toggle="tooltip" data-placement="left" title="{% trans "Not enough answers were given to publish the results." %}">
-                                                {{ course.name }}
-                                            </span>
+                                        {% if course.state == 'published' and not course|can_publish_grades %}
+                                            <span data-toggle="tooltip" data-placement="top" class="fa fa-lock" title="{% trans 'Not enough answers were given to publish the results.' %}"></span>
                                         {% endif %}
+                                        {{ course.name }}
```

Comment:
please check whether underlining the name when the row is hovered feels right

Reply:
why? it's consistent with highlighting the buttons when hovering over the row.

Reply:
Okay, I misinterpreted you there - i thought you meant all three pages. You're right, it would be more consistent.

On the other hand, with just the look and feel in mind, I feel like the color and cursor change are enough changes so the user understands that he is hovering a link.  For me, underlining the text feels wrong / too much, especially if the cursor is somewhere on the right part of the column but the text on the left is still underlined.

But, as I say, I agree on the consistency, so I'll leave it open to you and @janno42 to decide.

Reply:
they are now underlined